src/elements/inputText/inputText.tsx

```diff
@@ -11,10 +11,12 @@ interface IInputTextProps {
   field: FormField;
   icon: IconProp;
   text: string;
-  onChange: (value: React.SetStateAction<string>) => void;
+  onChange: ((value: React.SetStateAction<string>) => void) | ((value: string) => void);
 }
 
-export function createChangeProcessor(setter: (value: React.SetStateAction<string>) => void) {
+export function createChangeProcessor(
+  setter: ((value: React.SetStateAction<string>) => void) | ((value: string) => void)
+) {
   const changeProcessor = (e: FormEvent) => {
     setter((e.target as HTMLInputElement).value);
   };
```

Comment:
![image](https://user-images.githubusercontent.com/25075672/147213495-ea1d4092-c14d-410c-a981-83910e6cbb35.png)

Reply:
made as you said